### PR TITLE
add mesh to traffic-templates

### PR DESCRIPTION
**Pull request by kalantar**
- add mesh gateway to traffic-templates
- also remove lingering references to hub repo

### CODE REVIEW

**Review by Alan-Cha — DISMISSED** [review]

**Comment by kalantar** [comment]
failing tests are known; are related to https://github.com/iter8-tools/iter8/issues/1460

File: charts/iter8/templates/_task-github.tpl [hunk 1]
```diff
@@ -23,6 +23,6 @@
     headers:
       Authorization: token {{ .token }}
       Accept: application/vnd.github+json
-    payloadTemplateURL: {{ default "https://raw.githubusercontent.com/iter8-tools/hub/iter8-0.12.1/templates/_payload-github.tpl" .payloadTemplateURL }}
+    payloadTemplateURL: {{ default "https://raw.githubusercontent.com/iter8-tools/iter8/iter8-0.13.7/templates/notify/_payload-github.tpl" .payloadTemplateURL }}
```
Comment (Alan-Cha) [inline charts/iter8/templates/_task-github.tpl:26]: Should this be pointing to `0.13.7`?

**Review by Alan-Cha — APPROVED** [review]

**Review by sriumcp — APPROVED** [review]
lgtm

File: charts/iter8/templates/_task-github.tpl (hunk 1 above)
Comment (Alan-Cha) [reply charts/iter8/templates/_task-github.tpl:26]: There is now an iter8 0.14.0 release https://github.com/iter8-tools/iter8/releases/tag/iter8-0.14.0

**Pull request merged**